### ISSUE TITLE
LUKS functionality doesn't work at all on Windows 10

### ISSUE THREAD

**Issue by alexskc** [t-d-k/LibreCrypt#57]
This was actually working fine a few hours ago for me, but after restarting my computer, it suddenly stopped working. I can't mount anything LUKS, I can't create anything LUKS. Any attempts simply result in a "Unable to open container. Please check your keyphrase and settings, and try again." I've tried uninstalling and reinstalling and installing and reinstalling FreeOTFE and it just refuses to work.

**Comment by alexskc** [comment]
After a lot of troubleshooting, I found the issue. LUKS doesn't work when the Windows non-unicode locale is set to Japanese. Might be the case with other languages, I haven't tested.

**Comment by t-d-k** [comment]
Does the password contain non-ASCII characters? If it does, this is a known limitation - see the Readme.

**Comment by alexskc** [comment]
Nope. No LUKS functionality works at all. Neither my actual LUKS partition, nor the luks.box test file. But when I change the system's locale back to en-US, and restart my computer, everything works exactly as expected.
